tests/testInterp.py

```diff
@@ -33,14 +33,8 @@
 import lsst.afw.display.ds9 as ds9
 import lsst.meas.algorithms as algorithms
 import lsst.meas.algorithms.defects as defects
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace_setVerbosity("algorithms.Interp", verbose)
```

Comment:
I think it'd be good to do the same thing here that you do in testPsfDetermination: set logging to INFO or DEBUG or TRACE with comment on how to increase log level.